src/ApiCall.js

```diff
@@ -0,0 +1,254 @@
+                            })
+                        }
+                    </ul>
+                    {
+                    showMore && !showSaved ? <p className='more' onClick={ handleClick } >More</p> : null
+                    }
+                    {/* <p onClick={ handleClick }> {recipes ? 'more': ''}</p> */}
+                </div>
+                : <div className="recipeCard">
+                    {
+                        showSaved
+                        ? savedRecipes.map( (savedRecipe) => {
+                            return(
+                                savedRecipe.key === display 
+                                ? <Recipe 
+                                    key={savedRecipe.key}
+                                    title={savedRecipe.data[0].label}
+                                    featured={savedRecipe.data[0].source}
+                                    time={savedRecipe.data[0].totalTime}
+                                    imgSorce={savedRecipe.data[0].image}
+                                    ingredients={savedRecipe.data[0].ingredientLines}
+                                    url={savedRecipe.data[0].url}
+                                    cuisine={savedRecipe.data[0].cuisineType}
+                                    meal={savedRecipe.data[0].mealType}
+                                    dish={savedRecipe.data[0].dishType}
+                                    handleClickBack={ getBackClick }
+                                    handleClickRemove= { getRemoveClick }
+                                    remove={savedRecipe.key}
+                                />
+                                : null
+                            )
+                        })
+                        : recipes.map( (recipe) => {
+                            return(
+                                recipe.recipe.url === display 
+                                ? <Recipe 
+                                    key={recipe.recipe.url}
+                                    title={recipe.recipe.label}
+                                    featured={recipe.recipe.source}
+                                    time={recipe.recipe.totalTime}
+                                    imgSorce={recipe.recipe.image}
+                                    ingredients={recipe.recipe.ingredientLines}
+                                    url={recipe.recipe.url}
+                                    cuisine={recipe.recipe.cuisineType}
+                                    meal={recipe.recipe.mealType}
+                                    dish={recipe.recipe.dishType}
+                                    handleClickBack={ getBackClick }
+                                    handleClickSave= { getSaveClick }
+                                />
+                                : null
+                            )
+                        })
+                    }
+                </div>
+            }
+        </div>
```

Comment:
code makes sense logically throughout, but maybe there could be parts that you'd want to omit and try to make more DRY. Maybe think of making some more components to easily organize your code.